src/app/modules/form/admin-form/admin-form.payments.controller.ts

```diff
@@ -223,7 +229,16 @@ export const _handleUpdatePayments: ControllerHandler<
           level: PermissionLevel.Write,
         }),
       )
-      // Step 3: User has permissions, proceed to allow updating of start page
+      .andThen(checkFormIsEncryptMode)
+      // Step 3: Check that the payment form has a stripe account connected
+      .andThen((form) => {
+        if (form.payments_channel.channel !== PaymentChannel.Unconnected) {
+          return ok(form)
+        } else {
+          return err(new PaymentChannelNotFoundError())
+        }
```

Comment:
/very minor nit: reduce negation where possible

```suggestion
        if (form.payments_channel.channel === PaymentChannel.Unconnected) {
          return err(new PaymentChannelNotFoundError())
        }
        return ok(form)
```

Reply:
Thanks Tim! [023eca4](https://github.com/opengovsg/FormSG/pull/6258/commits/023eca4aaeca7d3a5ea6b1bd6fd985975a42723e)